### PR TITLE
update manylinux action

### PR DESCRIPTION
**Pull request by timosachsenberg**
@jpfeuffer is the "context" correct?

### CODE REVIEW

**Comment by jpfeuffer** [comment]
No, the dockerfiles for this action were not transfered yet.

**Comment by timosachsenberg** [comment]
k I can transfer them.

**Comment by timosachsenberg** [comment]
Is it a good idea to replicate the folder structure?
dockerfiles/pyopenms/manylinux/Dockerfile
as in the old Dockerfile repo?
or call it Dockerfile_pyopenms_on_manylinux or similar?

File: .github/workflows/deploy-manylinux-docker.yml [hunk 1]
```diff
@@ -47,7 +40,7 @@ jobs:
         uses: docker/build-push-action@v3
         with:
           push: true # Will only build if this is not here
-          context: dockerfiles/pyOpenMS/manylinux
+          context: dockerfiles
```
Comment (jpfeuffer) [inline .github/workflows/deploy-manylinux-docker.yml:43]: revert

Comment (timosachsenberg) [reply .github/workflows/deploy-manylinux-docker.yml:43]: Will do

Comment (timosachsenberg) [inline .github/workflows/deploy-manylinux-docker.yml:43]:
```suggestion
          context: dockerfiles/pyOpenMS/manylinux
```

**Pull request merged**